src/OrchardCore.Modules/OrchardCore.Tenants/Views/Admin/Create.cshtml

```diff
@@ -1,113 +1,24 @@
-                {
-                    <option value="@recipe.Name" selected="@(Model.RecipeName == recipe.Name)">
-                        @recipe.DisplayName
-                    </option>
-                }
-            </select>
-        </div>
+        <partial name="_TenantInfo" model="Model" />
     </div>
-    <h6>
-        @T["Database Presets"]
-        <span class="text-muted form-text small">@T["Optionally specify which presets should be used for the Setup experience."]</span>
-    </h6>
-    <div class="row">
-        @if (!Model.DatabaseConfigurationPreset)
-        {
-            <div class="mb-3 col-md-6">
-                <label asp-for="DatabaseProvider">@T["What type of database to use?"]</label>
-                <select asp-for="DatabaseProvider" class="form-select">
-                    <option value="" data-connection-string="false">@T["None"]</option>
-                    @foreach (var provider in DatabaseProviders)
-                    {
-                        <option value="@provider.Value" data-connection-string="@provider.HasConnectionString.ToString().ToLower()" data-table-prefix="@provider.HasTablePrefix.ToString().ToLower()" data-connection-string-sample="@T["The connection string to your database instance. e.g., {0}", provider.SampleConnectionString]">@provider.Name</option>
-                    }
-                </select>
-            </div>
-        }
-        <div class="mb-3 col-md-6 tablePrefix" asp-validation-class-for="TablePrefix">
-            <label asp-for="TablePrefix">@T["Table Prefix"]</label>
-            <span asp-validation-for="TablePrefix" class="text-danger"></span>
-            <input asp-for="TablePrefix" class="form-control" />
-        </div>
-    </div>
-    @if (!Model.DatabaseConfigurationPreset)
-    {
-        <div class="row connectionString" asp-validation-class-for="ConnectionString">
-            <div class="mb-3 col-md-12">
-                <label asp-for="ConnectionString">@T["Connection string"]</label>
-                <input asp-for="ConnectionString" class="form-control" />
-                <span asp-validation-for="ConnectionString" class="text-danger"></span>
-                <span id="connectionStringHint" class="text-muted form-text small"></span>
-            </div>
-        </div>
-    }
 
-    @if (Model.FeatureProfiles.Any())
-    {
-        <h6>
-            @T["Feature profile"]
-            <span class="text-muted form-text small">@T["Optionally specify which feature profile should be applied to this tenant."]</span>
-        </h6>
-        <div class="row" asp-validation-class-for="FeatureProfile">
-            <div class="mb-3 col-md-6">
-                <select asp-for="FeatureProfile" asp-items="Model.FeatureProfiles" class="form-select"></select>
-            </div>
-        </div>
-    }
+    <partial name="_TenantRecipes" model="Model" />
+    <partial name="_TenantDatabaseInfo" model="Model" />
+    <partial name="_TenantFeatureProfile" model="Model" />
```

Comment:
@jtkech not sure we need to worry about this since the Tenants admin used view/viewmodels there is not display driver being used. so if someone want to override the create/edit views, they would have to define their own fields anyway.

Reply:
Okay no problem, so just to say that in a theme, yes we can override a pure mvc view, and if this view trigger a shape with the tag helper, the theme can also override this shape even if not driven by a driver, here by shape type if no alternates was added, which can also be done in the original razor view (also in a liquid view).

Yes I was thinking about someone that just want to not render the new options in the form.

But no problem that's okay, I'm just not used to see `<partial>` in our razor views ;)